test/cocotb/osdtestlib/debug_interconnect.py

```diff
@@ -145,18 +225,17 @@ def receive_packet(self, dut, response):
             yield RisingEdge(dut.clk)
 
             if dut.debug_out.valid.value:
-
                 # extend the buffer with the debug module's output
                 response.append(dut.debug_out.data.value.integer)
-                
+
                 # if the last flit has been sent
                 if dut.debug_out.last.value:
-                    # stop reading in the response 
+                    # stop reading in the response
                     break
-                
-    @cocotb.coroutine        
-    def read_register(self, dut, destination, source, reg_width, address,
-                      response):
+
+    @cocotb.coroutine
+    def read_register(self, dut, destination, source, word_width, address,
+                      rx_packet):
         """Uses the subroutine 'send_packet' to read a value from
            a specified register and reads the response to tell the user if
            the read process was successful
```

Comment:
Don't explain *how* the method is implemented, but explain what it does for a user. So, instead of saying "uses subroutine xyz to do ..." say "Read a value from a specified register and return the read value to the user."